packages/ember-htmlbars/tests/integration/mutable_binding_test.js

```diff
@@ -111,7 +111,7 @@ QUnit.test('using a string value through middle tier does not trigger assertion'
 
   runAppend(view);
 
-  assert.strictEqual(bottom.attrs.stuff.value, 'foo', "precond - the data propagated");
+  assert.strictEqual(bottom.attrs.stuff, 'foo', "precond - the data propagated");
```

Comment:
This test was trolling me. It was incorrectly asserting that the non-stream value became a stream.